djangoappengine/guestbook/models.py

```diff
@@ -80,14 +81,15 @@ def put_greeting(self, content, author, user, title):
         greeting.content = content
         greeting.author = author
         cache_result = None
-        if greeting.put():
-            cache_result = memcache.delete(self.name)
-            if user:
-                taskqueue.add(
-                    method='GET',
-                    url='/mail',
-                    params={'title': title, 'author': author})
-            return cache_result
+        if (greeting.author == author):
```

Comment:
greeting.author = author already define above. therefore this condition already TRUE. Is it neccessary?